client/src/shared/components/IconButton/IconButton.tsx

```diff
@@ -1,10 +1,7 @@
 import style from './styles.module.scss';
+import { IconButtonProps } from '@/types/interfaces';
 import { FC } from 'react';
 
-interface IconButtonProps {
-  icon: string;
-}
-
 export const IconButton: FC<IconButtonProps> = ({ icon }) => {
```

Comment:
the interface IconButtonProps belongs to this component 
put it above component function

be consistent in nameing - no I (becase it seems you don't use it project-wise)